### DISCUSSION TITLE
功能建议：添加latex行内公式的渲染

### DISCUSSION THREAD

**Discussion by syw-robotics** [lollipopkit/flutter_gpt_box#6] [Ideas]
非常感谢作者提供的UI，相比于自己写的命令行交互体验有了质的飞跃；
目前UI不支持行内公式的渲染，如下：
![image](https://github.com/lollipopkit/flutter_gpt_box/assets/84261527/8953707d-b7be-433c-a550-ebd6397030ed)

希望可以有行内公式渲染的功能，目前个人无计算机背景，暂无能力参与开发，十分抱歉；

**Comment by lollipopkit** [comment]
![image](https://github.com/lollipopkit/flutter_gpt_box/assets/10864310/8de56b29-a7ef-4b82-96b3-d12bcf60c004)
已修复，抱歉不太了解latex

**Reply by syw-robotics** [reply]
十分感谢， best wish！
